includes/class-wc-payments-utils.php

```diff
@@ -355,4 +355,62 @@ public static function get_order_intent_currency( WC_Order $order ): string {
 	public static function set_order_intent_currency( WC_Order $order, string $currency ) {
 		$order->update_meta_data( self::ORDER_INTENT_CURRENCY_META_KEY, $currency );
 	}
+
+	/**
+	 * Returns a formatted message suitable for end-user notice
+	 *
+	 * @param string    $message Suggested message.
+	 * @param Throwable $e Exception to format.
+	 *
+	 * @return string
+	 */
+	public static function wc_format_api_exception_message( $message, $e ) {
+
+		if ( $e instanceof WCPay\Exceptions\API_Exception ) {
+			if ( 'wcpay_amount_too_low' === $e->get_error_code() ) {
+				return self::wc_format_amount_too_low_exception( $e );
+			}
+		}
+
+		return $message;
+	}
+
+	/**
+	 * Returns a formatted message for amount_too_low errors.
+	 *
+	 * @param API_Exception $e API_Exception instance to format.
+	 *
+	 * @return string User-friendly error message.
+	 */
+	public static function wc_format_amount_too_low_exception( $e ) {
```

Comment:
I'd actually go for a new exception type then and override `getMessage()` to format it accordingly. But this might require to introduce a factory method to create a proper exception based on error code.